android/src/main/java/com/swmansion/reanimated/layoutReanimation/SharedTransitionManager.java

```diff
@@ -85,19 +88,20 @@ protected void setNativeMethods(NativeMethodsHolder nativeMethods) {
     mNativeMethodsHolder = nativeMethods;
   }
 
-  private void startSharedTransitionForViews(List<View> sharedViews, boolean withNewElements) {
+  private boolean startSharedTransitionForViews(List<View> sharedViews, boolean withNewElements) {
```

Comment:
```suggestion
  private boolean tryStartSharedTransitionForViews(List<View> sharedViews, boolean withNewElements) {
```
To better emphasise the fact that animation may not be started, let's prepend the name with `try` or `maybe`.